pages/landing/hipaa/technical-details/index.html

```diff
@@ -69,4 +304,16 @@
   <div id="how-it-works" class="section section-dark">
     <div class="container">{% include_relative _how-it-works.html %}</div>
   </div>
+  <div id="more-app-details" class="section section-dark">
+    <div class="container">{% include_relative _app-more-details.html %}</div>
+  </div>
+  <div id="how-it-works" class="section section-dark">
+    <div class="container">{% include_relative _infra-how-it-works.html %}</div>
+  </div>
+  <div id="more-infra-details" class="section section-dark">
+    <div class="container">{% include_relative _infra-more-details.html %}</div>
+  </div>
+  <div id="more-infra-details" class="section section-dark">
+    <div class="container">{% include_relative _code-viewer-css-js.html %}</div>
+  </div>
```

Comment:
Tacking all this stuff on to the bottom of the app page is confusing and creates a page that's large and hard to follow. Since there are two links in the nav and two links from the "how it works" section of the home page anyway, these should be two separate pages: one for reference apps, one for the reference architecture.

Reply:
Roger - sounds we're going to go with the in-page tabbed navigation to address this.